heartbeat/galera

```diff
@@ -491,7 +491,7 @@ greater_than_equal_long()
 
 galera_to_pcmk_name()
 {
-    local galera=$1
+    local galera=${1%:*}
```

Comment:
Stripping a port number in this way would actually process incorrectly Galera nodes that are defined as '[IPv6-address]'. Such a format is also mishandled by the following `cluster_host_map` mapping code. Fixing that code instead to split a single mapping on the first ':' character should also allow specifying a port number in Galera node names.

Reply:
The situation of galera with IPv6 is a bit messy. Using IPv6 directly in gcomm or wsrep_listen_addr used to cause problem due to some version of galera not parsing IPv6 properly [1]. The situation is clean nowadays but for sure the galera_to_pcmk_name is another complication.

[1] https://github.com/codership/galera/issues/519